packages/react/src/box/styles.ts

```diff
@@ -478,6 +485,11 @@ export function boxStyles({
 				listStyle: 'none',
 				margin: 0,
 				padding: 0,
+				wordWrap:
```

Comment:
What about using ternaries?
```suggestion
				wordWrap: breakWords === true ? 'break-word' : breakWords === false ? 'normal' : undefined
```

Reply:
I see nested ternaries as being less readable. The grouping of `(a === b && 'x')` is easier to spot than `a === b ? 'x'`.

`(a === b && 'x')` is a complete self contained bit of code. `a === b ? 'x'` is an incomplete, invalid ternary statement. With the ternary method, it requires a bit more mental gymnastics to understand the logic of the code. You need to keep more context in your mind as you read through it.

It is also a linting rule in the BIIS products to not allow nested turneries mainly for the reduced code readability. I didn't set up the rule though I do agree with it.

I'm fine with ternaries, just not nested ternaries.

Reply:
Yep, fair enough